doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/demand-controlled-ventilation.tex

```diff
@@ -290,18 +290,31 @@ \subsection{Proportional Control}\label{proportional-control}
 {V_{ot - design,i}} = \frac{{\left( {{R_{p,i}}.{P_{z,i}} \times {SchedFrac_i}} \right) + ({R_{a,i}}.{A_{z,i}} \times {SchedFrac_i})}}{{{E_i}}}
 \end{equation}
 
+if ProportionalControlBasedOnDesignOARate is specified, the required intake flow rate will be revised as:
+
+ \begin{equation} {V{ot - design,i}} = \frac{Design Outdoor Air Flow Rate_i} {E_i} \end{equation}
```

Comment:
I know you just kept the pattern going so I'm not asking you to revamp the whole section, but I will lament that all these _i_ subscripts are not helpful.  Instead of saying "for an i_ith zone", and then including tons of those subscripts, we should just say "for a zone", and then clean the equations up.  Nothing for you to do here, just my nagging.